lib/constructs/interface.js

```diff
@@ -1194,17 +1194,25 @@ class Interface {
 
   generateIface() {
     this.str += `
-      function makeWrapper(globalObject) {
+      function makeWrapper(globalObject, newTarget) {
```

Comment:
Sorry for the extra work, but please rebase on https://github.com/jsdom/webidl2js/pull/240.